### PR TITLE
Capitalize Swift in FoundationShims.h

### PR DESCRIPTION
**Pull request by sanjeetsuhag**

### CODE REVIEW

File: stdlib/public/SwiftShims/FoundationShims.h [hunk 1]
```diff
@@ -21,9 +21,9 @@
 
 //===--- Layout-compatible clones of Foundation structs -------------------===//
 // Ideally we would declare the same names as Foundation does, but
-// swift's module importer is not yet tolerant of the same struct
+// Swift's module importer is not yet tolerant of the same struct
 // coming in from two different Clang modules
-// (rdar://problem/16294674).  Instead, we copy the definitions here
+// (rdar://problem/16294674). Instead, we copy the definitions here
```
Comment (gribozavr) [inline stdlib/public/SwiftShims/FoundationShims.h]: It is customary to use two spaces to separate sentences.

File: stdlib/public/SwiftShims/FoundationShims.h [hunk 2]
```diff
@@ -21,9 +21,9 @@
 
 //===--- Layout-compatible clones of Foundation structs -------------------===//
 // Ideally we would declare the same names as Foundation does, but
-// swift's module importer is not yet tolerant of the same struct
+// Swift's module importer is not yet tolerant of the same struct
```
Comment (gribozavr) [inline stdlib/public/SwiftShims/FoundationShims.h]: This part LGTM.

**Comment by gribozavr** [comment]
Please also use a more descriptive commit message.

File: stdlib/public/SwiftShims/FoundationShims.h (hunk 1 above)
Comment (coobersdoobers) [inline stdlib/public/SwiftShims/FoundationShims.h]: Where?

Comment (gribozavr) [inline stdlib/public/SwiftShims/FoundationShims.h]: There are multiple opinions on this subject, but one convention is to insert two spaces after a full stop.  Wikipedia summarizes viewpoints nicely: https://en.wikipedia.org/wiki/Sentence_spacing

Comment (coobersdoobers) [inline stdlib/public/SwiftShims/FoundationShims.h]: I see, but I don't think comments are best place for this convention, for me it doesn't feel right to see two sentences in one line, separated by too much space. Thanks for reply!

**Comment by tcyrus** [comment]
:+1:

**Comment by gribozavr** [comment]
Applied the capitalization in https://github.com/apple/swift/commit/3fb38535d8c0d766aa8521e8fd6e4d9267013629

**Pull request closed without merging**